universum/github_handler.py

```diff
@@ -59,16 +52,26 @@ def __init__(self, *args, **kwargs):
                     Please specify this parameter by using '--trigger-url' ('-tu')
                     command line parameter or by setting TRIGGER_URL environment variable.
                 """)
+        try:
```

Comment:
This whole block should be moved to utils as well.

Reply:
that's exactly what I've written in comment above. The question is: should it become a part of 'read' function? Should the function become 'read_and_check' or two separate functions?